examples/models/rbc_catastrophe.yaml

```diff
@@ -0,0 +1,90 @@
+name: Real Business Cycle
+
+model_type: dtcc
+
+symbols:
+
+   exogenous: [z, u]
```

Comment:
I think this is the same symbol (`u` ) as in the rewards section. We should fix this